src/shared/components/search-filters/search-filters-wrapper.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { Jurisdiction } from '../../domain';
+import { DefinitionsService } from '../../services';
+import { READ_ACCESS } from '../../domain/case-view/access-types.model';
+
+@Component({
+    selector: 'ccd-search-filters-wrapper',
+    template: `
+        <ccd-search-filters
```

Comment:
Can we have a template in a separate file to be consistent?

Reply:
Done.